client/internal/http/request.go

```diff
@@ -89,6 +89,8 @@ func (r *Request) send(output any) error {
 		return err
 	}
 
+	req.Header.Add("User-Agent", fmt.Sprintf("cdo_tf_provider/%s", Version))
```

Comment:
We actually already adding headers some lines down, can we add it there please? So that things are in one place. Something like this maybe:
```
func (r *Request) addHeaders(req *http.Request) {
	r.addAuthHeader(req)
	r.addOtherHeader(req)
	r.addJsonContentTypeHeaderIfNotPresent(req)
++	r.addUserAgentHeader(req)
}
```

Reply:
Good shout. Done